### PR TITLE
[#96986300] Smoke tests to on alert demo environment issue

### PR DESCRIPTION
**Pull request by mtekel**
Refactor of how smoketests are defined:
- create unified smoketest job template, with parametrized name, deploy environment, target platform, cron triggers and email address for notifications
- use this new template to define all smoketests:
  - smoketest-ci-aws
  - smoketest-ci-gce
  - smoketest-demo-aws
  - smoketest-demo-gce
- set demo smoketest run periodicity to 2 minutes, so that jobs run often and issue notification delay is minimised
### How to deploy on Jenkins

Because these jobs have different names that the current smoketest-aws and smoketest-gce, these two old jobs need to be removed manually. You would also need to remove the default dsl job, because it would re-create these smoketest jobs on ansible run. After you have removed these 3 jobs, subsequent ansible run will create a new default dsl job which will in turn create all the new smoketest jobs.

### CODE REVIEW

**Comment by dcarley** [comment]
The code looks :+1: but I haven't been able to test this and I'm not able to do the deployment right now. Could someone else merge and see it out?

File: site.yml [hunk 1]
```diff
@@ -105,34 +104,50 @@
...
       copy: src=jobs dest=/var/lib/jenkins/jobs/default-dsl-job/workspace directory_mode=0750 mode=0640 owner=jenkins group=jenkins
-    - name: create jenkins job directory structure
-      action: file path={{item}} state=directory mode=0775 owner="jenkins"
-      with_items:
-        - /var/lib/jenkins/jobs/default-dsl-job/workspace/jobs/terraform-deploy
-        - /var/lib/jenkins/jobs/default-dsl-job/workspace/jobs/ansible-deploy-aws
-        - /var/lib/jenkins/jobs/default-dsl-job/workspace/jobs/ansible-deploy-gce
-        - /var/lib/jenkins/jobs/default-dsl-job/workspace/jobs/smoke-test-aws
-        - /var/lib/jenkins/jobs/default-dsl-job/workspace/jobs/smoke-test-gce
-        - /var/lib/jenkins/jobs/default-dsl-job/workspace/jobs/demo-env-deploy
```
Comment (dhilton) [inline site.yml:116]: This job seems to be missing in the revised ansible playbook? Any ideas?

**Comment by mtekel** [comment]
Added the demo job back in.

File: site.yml [hunk 2]
```diff
@@ -21,7 +21,6 @@
       - job-dsl
       - github-api
       - github-oauth
-      - parameterized-trigger
```
Comment (dhilton) [inline site.yml:24]: This broke https://github.com/alphagov/multicloud-deploy/pull/26 can we add back in?

**Pull request merged**